client/lib/ffi.rs

```diff
@@ -8,15 +8,13 @@ use std::{
     sync::Mutex,
 };
 
-use lazy_static::lazy_static;
 use tokio::runtime;
 
 use super::error::{Error, Result};
+use once_cell::sync::Lazy;
```

Comment:
Should be with other 3rd party crate imports.

Reply:
Moved.